templates/error_page.hbs

```diff
@@ -10,12 +9,41 @@
     <h2>{{t 'not_authorized'}}</h2>
   {{/is}}
 
-  {{#is error 'not_found'}}
-    <h2>{{t 'nonexistent_page'}}</h2>
-    <p>{{t 'mistyped_address_or_moved_page'}}</p>
-  {{/is}}
-
-  {{#link 'help_center'}}
-    {{t 'back_to_homepage'}}
-  {{/link}}
+{{#is error 'not_found'}}
+  <h1 class="page-title compact-bottom">404</h1>
+  <h2 class="compact-top">Page not found</h2>
+  <p>The link you followed may be broken or we may have moved the page. If you cannot find what you are looking for, <a href="https://www.artstor.org/contact-us">contact us</a>.
+  <p>
+  	<a href="/" class="button">Back to Homepage</a>
+  	<a href="http://support.artstor.org/" class="button hollow" target="_blank">Get Help</a>
+  </p>
+<div>
+  <img class="fit" src="https://theme.zdassets.com/theme_assets/1006386/1afd840bddc97c7702e5d701be71eba177e0a776.jpg" alt="Philippe Mercier, The Sense of Sight, 1744 – 1747. Yale Center for British Art">
```

Comment:
let's resize the image and remove the fit class

Reply:
@anjanette got it